models/import.js

```diff
@@ -1,5 +1,285 @@
+const DateString = Match.Where(function (dateAsString) {
+  check(dateAsString, String);
+  return moment(dateAsString, moment.ISO_8601).isValid();
+});
+
+class TrelloCreator {
+  constructor() {
+    // the object creation dates, indexed by Trello id (so we only parse actions once!)
+    this.createdAt = {
+      board: null,
+      cards: {},
+      lists: {},
+    };
+    // the labels we created, indexed by Trello id (to map when importing cards)
+    this.labels = {};
+    // the lists we created, indexed by Trello id (to map when importing cards)
+    this.lists = {};
+    // the comments, indexed by Trello card id (to map when importing cards)
+    this.comments = {};
+  }
+
+  checkActions(trelloActions) {
+    check(trelloActions, [Match.ObjectIncluding({
+      data: Object,
+      date: DateString,
+      type: String,
+    })]);
+    // XXX perform deeper checks based on type
+  }
+
+  checkBoard(trelloBoard) {
+    check(trelloBoard, Match.ObjectIncluding({
+      closed: Boolean,
+      labels: [Match.ObjectIncluding({
+        // XXX check versus list
```

Comment:
It means as an extra security, we could check that these data (color, permission level, background color) are part of a specific list - because if they are not, we won't be able to import them.
But I also think it's a "nice to have" thing.